code/src/components/TodoItem.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react';
+
+import { useDispatch } from 'react-redux';
+import moment from 'moment';
+import styled from 'styled-components';
+import { todos } from '../reducers/todos';
+import { TrashBin } from '../lib/Image';
+import { Span } from '../lib/Text';
+import CustomCheckbox from '../lib/CustomCheckbox';
+
+const TodoItem = ({ item }) => {
+  const [active, setActive] = useState('');
+  const dispatch = useDispatch();
+
+  const handleRemoveTodo = () => {
+    dispatch(
+      todos.actions.removeTodo({
+        id: item.id,
+      })
+    );
+  };
+  
+  const toggleComplete = () => {
+    dispatch(todos.actions.setComplete(item.id));
+  };
+
+  const toggleAccordion = () => {
+    setActive(active === '' ? 'active' : '');
+  };
+
+  return (
+    <Container>
+      <Todo className={`accordion ${active}`} onClick={toggleAccordion}>
+        <Label>
+        <CustomCheckbox
+          checked={item.complete ? true : ""}
+          onChange={toggleComplete}
+        ></CustomCheckbox>
```

Comment:
I think you're missing a tab here after the Label tag

Reply:
Fixed 😄